### ISSUE TITLE
Multiple clients in browser-client code

### ISSUE THREAD

**Issue by gowthm-r7** [KoljaB/RealtimeSTT#83]
Hey there,
I'm using example_browserclient code by having seperate server and client pc (server.py code in server pc and index.html and client.js code in the client pc). I established the connection by connecting both the PCs in the same internet and hosting the server via server PC's IP address (instead of hosting it to localhost). It is working perfectly.

Can I handle multiple clients? Cos, I tried connecting another client and gave input to it, but got the same output in both the clients, eventhough the other client was set idle and given no input.

**Comment by KoljaB** [comment]
This is not implemented yet, so it can't handle multiple clients. RealtimeSTT can't do multiple transcriptions in parallel, so for this scenario some way of queueing the requests has to be implemented.

**Comment by gowthm-r7** [comment]
in what way? should the other clients wait for the previous client to finish?

**Comment by KoljaB** [comment]
Yes. Or you need to use multithreading or multiprocessing and probably use multiple gpus. This is due to how the dependent libraries are implemented, can't do much here.

**Comment by gowthm-r7** [comment]
Hey there.
I tried calling the audiototextrecorder class separately and independently for each and every client connection. Transcription happens with no issues, and the clients too get only their respective transcription, with no interruptions between the connected clients.
But, I'm noticing a huge utility of gpu, which is expected, as I'm calling the classes more than once and giving more load to the gpu.
So, to resolve this, queueing the requests in some way, as you said, is the best solution ig, so that I could run it in my installed gpu only smoothly.
Is there any possibility to make it happen in the existing realtimeSTT? after all these updates in this year?

**Comment by gowthm-r7** [comment]
Hey, 
You there?
I'm waitin for your reply.
Can you please respond?

**Comment by KoljaB** [comment]
I'm currently too busy to do anything for my OSS libraries. This might change in some weeks, I can't tell currently. Please don't expect any major changes or updates for this month.
